cmd/metrictank/metrictank.go

```diff
@@ -148,26 +152,26 @@ func main() {
 	// workaround for https://github.com/grafana/grafana/issues/4055
 	switch logLevel {
 	case 0:
```

Comment:
this is a breaking change that many people may inadvertently run into. eg an old level 1 changes from debug to fatal?
if we're going to break, might as well switch to strings in the config (like `level = debug`), that way we make sure people update their configs as needed, and it's more elegant.
does logrus have an elegant way to parse a string level declaration to an actual level variable?

I would like to get rid of this whole code section

Reply:
Yes, logrus can parse a debug level from a string. I'll see what I can do here as well